debian/control

```diff
@@ -53,7 +53,6 @@ Build-Depends: cmake (>= 3.5),
                libssl-dev,
                libtool,
                libudev-dev,
-               libnl-3-dev,
```

Comment:
I didn't get this part. Isn't it needed so on debian/ubuntu that package gets brought in automatically like it is for rpm based systems?

Reply:
Ignore this comment. I see it in the git commit.